python/cudf/cudf/core/series.py

```diff
@@ -5009,12 +5009,67 @@ def _get_dt_field(self, field):
         )
 
     def ceil(self, field):
+        """
+        Perform ceil operation on the data to the specified freq.
+        Parameters
+        ----------
+        field : str
+            One of ["D", "H", "T", "S", "L", "U", "N"]
+            See `frequency aliases <https://pandas.pydata.org/docs/\
+                user_guide/timeseries.html#timeseries-offset-aliases>`_
+            for more details on these aliases.
+        Returns
+        -------
+        Series
+            Series with the same index for a Series.
+        Examples
+        --------
+        >>> import cudf
+        >>> t = cudf.Series(["2001-01-01 00:04:45", "2001-01-01 00:04:58",
+        ... "2001-01-01 00:05:04"], dtype="datetime64[ns]")
+        >>> t.dt.ceil("T")
+        0   2001-01-01 00:05:00
+        1   2001-01-01 00:05:00
+        2   2001-01-01 00:06:00
+        dtype: datetime64[ns]
+        """
         out_column = self.series._column.ceil(field)
 
         return Series(
             data=out_column, index=self.series._index, name=self.series.name
         )
 
+    def floor(self, field):
+        """
+        Perform floor operation on the data to the specified freq.
+        Parameters
+        ----------
+        field : str
+            One of ["D", "H", "T", "S", "L", "U", "N"]
+            See `frequency aliases <https://pandas.pydata.org/docs/\
+                user_guide/timeseries.html#timeseries-offset-aliases>`_
+            for more details on these aliases.
+        Returns
+        -------
+        Series
+            Series with the same index for a Series.
+        Examples
+        --------
```

Comment:
```suggestion
        Perform floor operation on the data to the specified freq.

        Parameters
        ----------
        field : str
            One of ["D", "H", "T", "S", "L", "U", "N"]
            See `frequency aliases <https://pandas.pydata.org/docs/\
                user_guide/timeseries.html#timeseries-offset-aliases>`_
            for more details on these aliases.

        Returns
        -------
        Series
            Series with the same index for a Series.

        Examples
        --------
```

Reply:
Hey @galipremsagar, what exactly is the suggested change here?

Reply:
New lines are missing between sections.